pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -896,3 +901,30 @@ def test_parse_delimited_date_swap(all_parsers, datestring,
     result = parser.read_csv(StringIO(datestring), header=None,
                              dayfirst=dayfirst, parse_dates=[0])
     tm.assert_frame_equal(result, expected)
+
+
+gen_random_datetime = st.dates(
+    min_value=date(1000, 1, 1),
+    max_value=date(9999, 12, 31)
+)
+_DEFAULT_DATETIME = datetime(1, 1, 1).replace(hour=0, minute=0,
+                                              second=0, microsecond=0)
```

Comment:
`datetime(1, 1, 1)` is sufficient, as the default values for time components is already `0`.

Reply:
ok, changed this